src/app/pages/apps/store/apps-store.service.ts

```diff
@@ -92,20 +93,15 @@ export class AppsStore extends ComponentStore<AppsState> {
           };
         });
       }),
-      catchError(() => {
-        this.handleError();
+      catchError((error) => {
```

Comment:
Don't think removing individual error handling for those methods is better here. These methods are handling success case by mutating state. They should also handle error cases individually. Although, they are not called individually right now, so I can approve this, but if they are called individually in the future, it is assumed that they will handle all cases on their on because they are handling success case.